### PR TITLE
Bookstore: initialize project with components

### PR DESCRIPTION
**Pull request by Atifzada**
### Following changed have been made to this PR:

- Setup a new React application using [Create React App](https://reactjs.org/docs/create-a-new-react-app.html#create-react-app)
- Created a directory for your reusable components:` /components.`
- Installed [React Router V6](https://reactrouter.com/en/main/start/tutorial#setup)
- [Setup router](https://reactrouter.com/en/main/start/tutorial#adding-a-router) inside of `<App>`
- Added 2 routes in the application, each rendered a page component:
```
- /
- /categories
```

1. Created components inside of the` /components` directory:
2. Contained individual book state, as well as a button to delete the book
3. Contained a list to render individual books
4. Contained a form to create new books
5. Contained navigation to link to the 2 routes you created

- Made sure no linter errors are there

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,24 +1,18 @@
...
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <>
+      <Nav />
+      <Routes>
+        <Route path="/" element={<BooksList />} />
+        <Route path="/catergories" element={<p>Catergory</p>} />
```
Comment (tufoinnkuo10) [inline src/App.js]: - [x] Great job so far. Kindly, create a Category component and navigate this URL to the category component to fulfill the page. You can simply write 'under construction in it'. Nice work.
<Route path="/categories" element={<Categories />} />

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @Atifzada 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Gitflow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ components well added

Required Changes ♻️
Check the comments under the review

tufoin

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: src/App.js (hunk 1 above)
Comment (Atifzada) [reply src/App.js]: Thank you. Surely I will come up with solution

**Review by amejid — APPROVED** [review]
Hi @Atifzada, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Linter checks passing :heavy_check_mark: 
- Correct workflow :heavy_check_mark: 
- React router used for navigation :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**